packages/loopring_v3/circuit/Gadgets/MathGadgets.h

```diff
@@ -1719,6 +1744,11 @@ struct SelectorGadget : public GadgetT
     }
 };
 
+// if selector=[1,0,0] and values = [a,b,c], return a
+// if selector=[0,1,0] and values = [a,b,c], return b
+// if selector=[0,0,1] and values = [a,b,c], return c
+// special case,
+// if selector=[0,0,0] and values = [a,b,c], return a
```

Comment:
As the selector we expect the output of the `SelectorGadget`, so the special case should never trigger, but if that works very much not intended. :)